### PR TITLE
Remove assets from the foundry

### PR DESCRIPTION
**Pull request by somniumism**
- [x] Remove the types related to the asset from `types`
    - TransferAsset
        - struct AssetTransferInput
        - struct AssetOutPoint
        - struct AssetTransferOutput
    - struct AssetMintOutput    
    - struct AssetWrapCCCOutput
- [x] Remove assets from MemPoolMinFees in `miner` and `rpc types`
- [x] Remove assets from the e2e tests
- [x] Modify the e2e tests

resolves issue #7

### CODE REVIEW

**Comment by somniumism** [comment]
@HoOngEe I added you as a reviewer. There may be some parts that I forgot to remove. Therefore, please clone this PR and review carefully : )

I didn't remove `AssetTransaction` in `test/src/sdk/core/Transaction.ts` and the logic related to it, because It seems necessary for the transactions. Please check this part, too!

**Comment by kseo** [comment]
@somniumism this PR doesn't need to be complete. You can create following PRs if you missed something in this PR.

File: vm/src/executor.rs [hunk 1]
```diff
@@ -77,10 +73,12 @@ impl From<HashingError> for RuntimeError {
 struct Item(Vec<u8>);
 
 impl Item {
+    #[allow(dead_code)]
```
Comment (HoOngEe) [inline vm/src/executor.rs]: I think you'd better to write higher level warning suppression.
You can write `#[allow(dead_code)]` right above the mod executor in `vm/src/lib.rs` or `#![allow(dead_code)]` for crate level suppression. https://doc.rust-lang.org/rust-by-example/attribute.html

**Comment by HoOngEe** [comment]
@somniumism We can remove the `AssetTransaction` also. We do not manage asset layer any longer. Let's remove it. Other parts are looking good

File: vm/src/executor.rs (hunk 1 above)
Comment (somniumism) [reply vm/src/executor.rs]: Thank you for your good idea! I modified it : ))

**Pull request merged**